### PR TITLE
Reworked LocalizationManager to load data async

### PR DESCRIPTION
**Pull request by Bond-009**
I also replaced the hard coded json string with the already included resource.

### CODE REVIEW

File: Emby.Server.Implementations/Localization/LocalizationManager.cs [hunk 1]
```diff
@@ -159,29 +150,20 @@ private void LoadRatings(string country, ParentalRating[] ratings)
...
-        private List<string> GetRatingsFiles(string directory)
-        {
-            return _fileSystem.GetFilePaths(directory, false)
-                .Where(i => string.Equals(Path.GetExtension(i), ".txt", StringComparison.OrdinalIgnoreCase))
-                .Where(i => Path.GetFileName(i).StartsWith("ratings-", StringComparison.OrdinalIgnoreCase))
-                .ToList();
-        }
+        private IEnumerable<string> GetRatingsFiles(string directory)
+            => _fileSystem.GetFilePaths(directory, false)
+                .Where(i => string.Equals(Path.GetExtension(i), ".csv", StringComparison.OrdinalIgnoreCase))
```
Comment (JustAMan) [inline Emby.Server.Implementations/Localization/LocalizationManager.cs]: Why rename `.txt` to `.csv`? Just curious.

File: Emby.Server.Implementations/Localization/LocalizationManager.cs [hunk 2]
```diff
@@ -288,12 +268,7 @@ public ParentalRating[] GetParentalRatings()
                 countryCode = "us";
             }
 
-            var ratings = GetRatings(countryCode);
-
-            if (ratings == null)
-            {
-                ratings = GetRatings("us");
-            }
+            var ratings = GetRatings(countryCode) ?? GetRatings("us");
```
Comment (JustAMan) [inline Emby.Server.Implementations/Localization/LocalizationManager.cs]: maybe just 
```suggestion
           return GetRatings(countryCode) ?? GetRatings("us");
```

File: Emby.Server.Implementations/Localization/LocalizationManager.cs (hunk 1 above)
Comment (Bond-009) [reply Emby.Server.Implementations/Localization/LocalizationManager.cs]: They are csv this and way my editor shows them properly :p

**Comment by joshuaboniface** [comment]
@Bond-009 Looks like two build failures:

```
Localization/LocalizationManager.cs(65,39):  error CS0103: The name 'localizationPath' does not exist in the current  context  [/drone/src/Emby.Server.Implementations/Emby.Server.Implementations.csproj]
Localization/LocalizationManager.cs(273,20): error CS0103: The name 'ratings' does not exist in the current context [/drone/src/Emby.Server.Implementations/Emby.Server.Implementations.csproj] 
```

**Comment by Bond-009** [comment]
@joshuaboniface Fixed

**Pull request merged**